### DISCUSSION TITLE
etcd watch on trigger key

### DISCUSSION THREAD

**Discussion by dreamingdeer** [cybertec-postgresql/vip-manager#196] [Ideas]
https://github.com/dreamingdeer/vip-manager/commit/95bca1e495ac3a1e4bbde9935d6830ad76aca3ce

May be this better than loop with get ? 
Watch on trigger key if they changed change state ?

**Comment by pashagolub** [comment]
Yes, probably we should switch to that interface

**Comment by pashagolub** [comment]
Closed by #199
